src/claim/main.c

```diff
@@ -10,15 +10,18 @@
 #include <signal.h>
 
 #include "main.h"
+#include "claim.h"
```

Comment:
This causes a compilation error on windows
```
In file included from /home/stelios/netdata/src/claim/main.c:13:
/home/stelios/netdata/src/claim/claim.h:6:10: fatal error: daemon/common.h: No such file or directory
    6 | #include "daemon/common.h"
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
This was a quick test done yesterday, I am proceeding with this now.